src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/Components/TCEvaluationImageCells/index.js

```diff
@@ -34,7 +34,8 @@ class TCEvaluationImageCells extends Component {
            onMouseLeave={this.offHoverState.bind(this)}>
         <div className="TCEvaluationImageCells">
           <img width={30}
-               src={"data:image/"+this.props.value.format+";base64,"+this.props.value.data} />
+               src={"data:image/"+this.props.value.format+";base64,"+this.props.value.data} 
+	       alt="Can't Find" />
```

Comment:
Let's leave the alt text blank (empty string), since there isn't a good alternate explanation for the contents of the image -- I don't think "Can't Find" is quite correct, since for a user with a screen reader, the image is found, it's just that they can't see it.